src/data/set/lattice.lean

```diff
@@ -1285,16 +1315,16 @@ hf.infi_comp g
 end surjective
 end function
 
-/-! ### Disjoint sets -/
+/-! ### Disjoint sets
```

Comment:
When the module docstring is longer than one line, the heading should be on its own line, otherwise the markdown parser can get confused:
```suggestion
/-!
### Disjoint sets
```